index.js

```diff
@@ -80,16 +98,20 @@ function ensureProperties (obj) {
     try {
       for (var prop in obj) {
         if (!Object.prototype.hasOwnProperty.call(obj, prop)) continue
+        if (isDescendent(filterPaths, path.join('.')) && shouldFilter(filterKeys, prop)) {
+          result[prop] = '[FILTERED]'
```

Comment:
We have all caps `FILTERED` here vs  `Circular` above, should they use consistent casing?